core/services/remoraCameraManager/remoraCameraManager.py

```diff
@@ -0,0 +1,495 @@
+            "info",
+            "-thread_queue_size",
+            "512",
+            "-f",
+            "v4l2",
+            "-input_format",
+            input_format,
+            "-video_size",
+            res,
+            "-framerate",
+            str(fps),
+            "-i",
+            device,
+        ]
+
+        # Always end up in yuv420p for H.264 encoders
+        cmd += ["-vf", "format=yuv420p"]
+
+        if use_hw:
+            print("h264_v4l2m2m encoder selected")
+            cmd += [
+                "-c:v",
+                "h264_v4l2m2m",
+                "-pix_fmt",
+                "yuv420p",
+                "-b:v",
+                bitrate,
+                "-maxrate",
+                str(int(kbitrate) * 1200),
+                "-bufsize",
+                str(int(kbitrate) * 1500),
+                "-g",
+                str(fps),
+            ]
+        else:
+            print("libx264 encoder selected")
+            cmd += [
+                "-c:v",
+                "libx264",
+                "-preset",
+                preset,
+                "-profile:v",
+                "high",
+                "-pix_fmt",
+                "yuv420p",
+                "-x264-params",
+                f"scenecut=0:keyint={fps}:min-keyint={fps}",
+                "-b:v",
+                bitrate,
+                "-maxrate",
+                str(int(kbitrate) * 1200),
+                "-bufsize",
+                str(int(kbitrate) * 2000),
+                "-g",
+                str(fps),
+                "-bf",
+                "0",
+            ]
+        return cmd
+
+    def start_stream(self, camera: str) -> str:
+        """Start streaming for the specified camera."""
+        if camera == "front":
+            cam_config = self.settings_manager.settings.camera.FRONT
+        elif camera == "back":
+            cam_config = self.settings_manager.settings.camera.BACK
+        else:
+            raise ValueError(f"Camera '{camera}' not found in configuration.")
+
+        device = cam_config.device
+        use_hw = self.settings_manager.settings.camera.USE_HW_ENC
+
+        if device is None:
+            raise ValueError(f"Device not defined for camera '{camera}'.")
+
+        cmd = self.ffmpeg_cmd(cam_config, bool(use_hw))
+
+        if self.check_mediamtx_running() is False:
+            print("MediaMTX server is not running. Might cause streaming issues.")
+
+        # RTSP output
+        rtsp_url = f"rtsp://{self.settings_manager.settings.camera.IP}:{self.settings_manager.settings.camera.PORT}/{cam_config.name}"
+        cmd += [
+            "-f",
+            "rtsp",
+            "-rtsp_transport",
+            "tcp",
+            "-muxdelay",
+            "0.1",
+            rtsp_url,
+        ]
+
+        print(f"Starting stream for camera '{camera}' with command: {' '.join(cmd)}")
+
+        # Start the ffmpeg process
+        # pylint: disable=consider-using-with
+        proc = subprocess.Popen(cmd)
```

Comment:
I think this could fill with a lot of data, especially if there are problems. I think we should `devNULL` the stdout and stderr, presuming we don't want to handle.
```suggestion
        proc = subprocess.Popen(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```